manifests/params.pp

```diff
@@ -7,88 +7,10 @@
 #
 class simp_grafana::params {
 
-  $trusted_nets = simplib::lookup('simp_options::trusted_nets', { 'default_value' => ['127.0.0.0/8'] })
-  $firewall     = simplib::lookup('simp_options::firewall', { 'default_value' => false })
-  $ldap         = simplib::lookup('simp_options::ldap', { 'default_value' => false })
-
-  $admin_pw = passgen('grafana')
-
-  $app_pki_dir             = '/etc/pki/simp_apps/grafana/x509'
-  $app_pki_key             = "${app_pki_dir}/private/${facts['fqdn']}.pem"
-  $app_pki_cert            = "${app_pki_dir}/public/${facts['fqdn']}.pub"
-
-  $base_dn = simplib::lookup('simp_options::ldap::base_dn', { 'default_value' => simplib::ldap::domain_to_dn() } )
-  $bind_dn = simplib::lookup('simp_options::ldap::bind_dn', { 'default_value' => "uid=%s,${base_dn}" } )
-  $bind_pw = simplib::lookup('simp_options::ldap::bind_pw', { 'default_value' => undef } )
-
-  $ldap_urls   = simplib::lookup('simp_options::ldap::uri', { 'default_value' => [''] } )
-  $ldap_url    = $ldap_urls[0]
-  $ldap_server = inline_template(
-    '<%= @ldap_url.match(/(([[:alnum:]][[:alnum:]-]{0,254})?[[:alnum:]]\.)+(([[:alnum:]][[:alnum:]-]{0,254})?[[:alnum:]])\.?/) %>'
-  )
-
   case $facts['osfamily'] {
```

Comment:
If our intention was to move away from params, we should remove the entire file and make sure this functionality ends up elsewhere